### ISSUE TITLE
Invalid tree output when using deviations

### ISSUE THREAD

**Issue by ihrasko** [PANTHEONtech/lighty-yang-validator#135]
**Describe the bug**
When we use command `java -jar ./target/lyv.jar -f tree -a models` with YANG models as in attachment we get the tree output:
```
  +--rw data-container
     +--rw list-data* [name]
        +--rw name       string
        +--rw year?       int64
        +--rw month?       string
```
Its means that we see also `year` and `month` leaves which were removed in `deviation.yang` model.

**Instead we expect to get output as follows**
```
  +--rw data-container
     +--rw list-data* [name]
        +--rw name    string
```

*The bug is present in all supported versions 16.2.0 and 17.0.0*

**Comment by ihrasko** [comment]
Fixed in version 17.0.1.

**Comment by ihrasko** [comment]
The used input YANG models: [models.tar.gz](https://github.com/PANTHEONtech/lighty-yang-validator/files/10219382/models.tar.gz)
